dl4j-cv-labs/src/main/java/global/skymind/solution/object_detection/AvocadoBananaDetector/dataHelpers/LabelImgXmlLabelProvider.java

```diff
@@ -0,0 +1,60 @@
+package global.skymind.solution.object_detection.AvocadoBananaDetector.dataHelpers;
+
+import org.apache.commons.io.FilenameUtils;
+import org.datavec.image.recordreader.objdetect.ImageObject;
+import org.datavec.image.recordreader.objdetect.ImageObjectLabelProvider;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LabelImgXmlLabelProvider implements ImageObjectLabelProvider {
```

Comment:
double check VocLabelProvider, is it another version of LabelImgXmlLabelProvider? if yes then stick to VocLabelProvider, wanna retire custom redundant class.

Reply:
Tested VocLabelProvider with fruits dataset, need to place all annotations into a dedicated "Annotations" folder 
![image](https://user-images.githubusercontent.com/51309327/73995668-45027d80-4994-11ea-86b2-c01319fa2e3d.png)